### PR TITLE
Package update

### PR DESCRIPTION
**Pull request by Jonno1809**
Updated package.json to latest working versions (the version that passes all tests, not necessarily the latest published version) and removed tildes to prevent unwanted updates. 

These updated modules will have to be added to Nexus 3 before merging so that we can use it on the network at work.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -14,6 +14,7 @@
 	<artifactId>wcomponents-parent</artifactId>
 	<version>1.5.3-SNAPSHOT</version>
 
+
```
Comment (marksreeves) [inline pom.xml]: This change is not warranted

File: wcomponents-examples/src/main/java/com/github/bordertech/wcomponents/examples/table/SimplePaginationTableWithSelectablesExample.java [hunk 2]
```diff
@@ -1,6 +1,5 @@
 package com.github.bordertech.wcomponents.examples.table;
 
-
```
Comment (marksreeves) [inline wcomponents-examples/src/main/java/com/github/bordertech/wcomponents/examples/table/SimplePaginationTableWithSelectablesExample.java]: This change is not warranted

File: wcomponents-examples/src/test/java/com/github/bordertech/wcomponents/examples/SimplePaginationTableWithSelectablesExample_Test.java [hunk 3]
```diff
@@ -160,6 +160,7 @@ public void testActionConstraintsAcrossPagesWithoutRefresh() {
 			driver.findElement(byWComponentPath(SELECT_BUTTON)).isEnabled());
 
 		wTableWebElement = getTable();
+
```
Comment (marksreeves) [inline wcomponents-examples/src/test/java/com/github/bordertech/wcomponents/examples/SimplePaginationTableWithSelectablesExample_Test.java]: This change is not warranted

File: wcomponents-theme/src/main/js/wc/ui/table/action.js [hunk 4]
```diff
@@ -58,6 +58,7 @@ define(["wc/dom/event",
 					totalSelected = currentSelected + otherSelected;
 
 					if ((min && totalSelected < min) || (max && totalSelected > max)) {
+
```
Comment (marksreeves) [inline wcomponents-theme/src/main/js/wc/ui/table/action.js]: This change is not warranted

**Comment by marksreeves** [comment]
@Jonno1809 I suggest closing this, throwing away those 46 commits and trying again from a clean base. The actual work to update the npm versions is all we need. The rest just produces unnecessary noise.

**Comment by Jonno1809** [comment]
@marksreeves No idea why or how they got there...will try again

**Pull request closed without merging**